bcauth/jinja2/oauth2_provider/application_detail.html

```diff
@@ -0,0 +1,26 @@
+{% extends "account/profile.html" %}
+{% block head_subtitle %} - {{ _("OAuth2 Client Application") }} - {{ _("Details") }}{% endblock %}
+{% block body_title %}{{ _("OAuth2 Client Application") }} - {{ _("Details") }}{% endblock %}
```

Comment:
l10n: The string composition for `head_subtitle` and `body_title` here prevents translators from reordering the elements. I haven't looked at the structure of how these are used enough to suggest solutions, but it's probably the case there are some standard ways of dealing with the needs here in an l10n friendly way.